server/lib/pbench/test/unit/test_process_tb.py

```diff
@@ -6,86 +6,82 @@
 from typing import List
 
 from pbench.process_tb import ProcessTb, Results
-from pbench.test import tarball, bad_tarball, fixture_path
+from pbench.test import tarball, bad_tarball, receive_dir
 
 
 class MockConfig:
     TS = "run-1970-01-02T00:00:00.000000"
 
+    def __init__(self, mappings: dict):
+        self.mappings = mappings
+
     def get(self, section: str, option: str) -> str:
-        return ""
+        assert (
+            section in self.mappings and option in self.mappings[section]
+        ), f"Unexpected configuration option, {section!r}/{option!r}, in MockConfig.get()"
+        return self.mappings[section][option]
 
 
 class TestProcessTb:
     def test_token(self, monkeypatch, test_logger):
-        def mock_get(self, section: str, option: str) -> str:
-            assert option in [
-                "pbench-receive-dir-prefix",
-                "put-token",
-            ], f"Unexpected configuration option, {section!r}/{option!r}, in mocked config.get()"
-            return fixture_path if option == "pbench-receive-dir-prefix" else ""
+        def mock_is_dir(self) -> bool:
+            return True
 
         expected_error_msg = (
             "No value for config option put-token in section pbench-server"
         )
-        monkeypatch.setattr(MockConfig, "get", mock_get)
+        mappings = {
+            "pbench-server": {
+                "pbench-receive-dir-prefix": receive_dir,
```

Comment:
This should probably be
```
                "pbench-receive-dir-prefix": str(receive_dir),
```
That is, the mapping should be `str` to `str`, and, currently, `receive_dir` is a `Path`.

Reply:
You should take a look at how you are using `receive_dir`:  it may make more sense to change it to be a string and then convert it to a `Path` where needed, rather than having it as a `Path` and converting it to a string in lots of places.